From 83aaf1a468c6c870b4fa0069eee3237967d2fd0b Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Fri, 23 Nov 2012 15:56:28 +0100
Subject: [PORT FROM MAIN] Add Stream Flags key to setParameters

BZ: 70126

Audio Route Manager needs to know the type of stream to take the
appropriate routing decision (eg direct Output, MP3 Offload).
It must be define as key and filled by the policy.

Feature: AudioComms
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: yes
Change-Id: Iabfc1add07df9a98a2eab63b9558f1f31e9d2b76
Orig-Change-Id: I5d2273971ba8d279f3bbda1db811a39e99d3357f
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
---
 include/hardware/audio.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 6317b0d..51c70a0 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -107,6 +107,9 @@ __BEGIN_DECLS
 /* Screen state */
 #define AUDIO_PARAMETER_KEY_SCREEN_STATE "screen_state"
 
+/* Stream Flags */
+#define AUDIO_PARAMETER_KEY_STREAM_FLAGS "stream_flags"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

